### ISSUE TITLE
Add DetermineCommitId to (IIssues)BuildServer

### ISSUE THREAD

**Issue by janniksam** [cake-contrib/Cake.Issues.Recipe#141]
Just a reminder. Maybe its already in progress?

----

Possible implementation for Azure Devops:

```cs
public string DetermineCommitId(ICakeContext context)
{
	if (context == null)
	{
		throw new ArgumentNullException(nameof(context));
	}

	return context.EnvironmentVariable("BUILD_SOURCEVERSION");
}
```

**Comment by pascalberger** [comment]
Why do you need this?

If you mean for file linking support this was implemented with #20 (but in this case you wouldn't need to call `DetermineCommitId` yourself).

**Comment by janniksam** [comment]
Thats exactly what I was looking for,. Maybe I have looked through the wrong branch.
